test/ci/CacheAndUpdateITK.sh

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env bash
+
+echo ${ExternalData_OBJECT_STORES}
+mkdir -p "${ExternalData_OBJECT_STORES}"
+
+echo "Using ITK repository: ${ITK_REPOSITORY_REMOTE:=https://github.com/InsightSoftwareConsortium/ITK.git}"
```

Comment:
It would be great to be able to use the same travis_dashboard.cmake and appveyor_dashboard.cmake but it might be difficult as some project might require more environment variables, different build matrix,... That is definitely something to look into it.